conan/mana/conanfile.py

```diff
@@ -11,17 +11,15 @@ class ManaConan(ConanFile):
     generators = 'cmake'
     url = "http://www.includeos.org/"
 
-    #def build_requirements(self):
-        #eventually
-        #self.build_requires("includeos/%s@%s/%s"%(self.version,self.user,self.channel))
     def build_requirements(self):
+        self.build_requires("libcxx/{}@{}/{}".format("7.0", self.user,self.channel))
```

Comment:
you can write it as self.build_requires("libcxx/7.0@{}/{}".format(self.user,self.channel)
However the most correct would be to have self.build_requires("includeos/{}@{}/{}) as includeos provides the headers
Even more correct would be if we had a package for includeos_headers which we could depend on ..